spec/helpstation/processors/by_key_fetcher_spec.rb

```diff
@@ -22,7 +22,7 @@
       expect(subject).to be_a(Substation::Response::Success)
       expect(subject.output).to eq(
         my_input: input[:my_input],
-        my_output: output
+        my_output: 'test and {:mock_service=>:mock_value}'
```

Comment:
I think there are 2 approaches to this. 

If we write out the env, then the env should be defined in this file locally. Otherwise it will always break when somebody changes the global env.

If we use global env, then it should not be written out.

Either way works for me